### ISSUE TITLE
A11y_Windows Terminal_Close Window_Color contrast: Color contrast ratio for 'Close all' button is less than 4.5:1 ratio

### ISSUE THREAD

**Issue by Amulya-95** [microsoft/terminal#5425]
GitHubTags: #A11y_WindowsTerminal; #WindowsTerminal; #A11yTCS; #A11yMAS; #Win10; #DesktopApp; #ColorContrast; #MAS1.4.3; #C+E-H; #A11ySev2;

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-tagott%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Windows Terminal
Version: 0.10.781.0
OS: Windows 10 (Version 1909, Build: 18363.778)

**Repro Steps:**
1.Hit the URL "aka.ms/terminal" and install "Windows terminal (Preview)" application
2.Open "Windows terminal (Preview)" application
3.Tab to multiple tabs
4.Use "Alt + F4" to close whole window and pop up will be displayed
5.Verify whether 'Close All" button color contrast ratio is greater or equal to 4.5:1

**Actual Result:**
Color contrast ratio for 'Close all' button is less than 4.5:1 ratio

**Expected Result:**
Color contrast ratio for 'Close all' button should be greater than or equal to 4.5:1 ratio

**User Impact:**
Low vision users will find difficulty, if the color contrast ratio is less than 4.5:1 ratio

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment:**
![Bug 5425_Color contrast ratio fails for 'Close All' button](https://user-images.githubusercontent.com/51435266/79754443-2ece5180-8335-11ea-9aff-ff104cc38b4d.png)

**Comment by zadjii-msft** [comment]
I'm pretty sure this is the same fundamental root as #5395 - these buttons are using the user-defined accent color. I'm not sure how we go about resolving this issue appropriately though.

**Comment by DHowett-MSFT** [comment]
> Turn on the high contrast mode

The screenshot of the application does not match any known configuration of "high contrast mode" on the Windows operating system. The "contrast analyser" application would appear to be high contrast if it were enabled on the operating system.

Is this bug report valid?

**Comment by DHowett-MSFT** [comment]
On high contrast mode, this dialog looks like:

![image](https://user-images.githubusercontent.com/14316954/79778415-8c3eb000-82ed-11ea-87fa-fcbb498f3670.png)

There is strong contrast between the button text and the button background, as is required.

**Comment by Amulya-95** [comment]
![Fixed Bug 5425](https://user-images.githubusercontent.com/51435266/81063131-e5653100-8ef4-11ea-909a-195c4a5181df.png)

Not able to reproduce the issue. color contrast ratio is equal to 4.5:1. Hence closing the bug.
